### PR TITLE
Python wheels support 

### PR DESCRIPTION
**Pull request by Earammak**
Added workflow file bld_wheels_and_upload.yml for python wheels support.
Added macro condition in setpu.py and ibm_db.c to support db2cli dlls for both 32-bit and 64-bit windows platform.
Updated readme.md
Updated ibm_db.c to fix the compilation issue for connection attribute lists for Zos.
Updated pyproject.toml, to build wheels for macos.
Fix for python issue #857
Updated install.md

### CODE REVIEW

File: .github/workflows/bld_wheels_and_upload.yml [hunk 1]
```diff
@@ -75,39 +83,49 @@ jobs:
...
       - name: Install Python dependencies
-        run: |
-          python -m pip install --upgrade pip build
+        run: python -m pip install --upgrade pip build
       - name: Build sdist
         run: python -m build --sdist --no-isolation
+      - name: Build source distribution 
+        run: |
+          cd dist
+          tar -xzvf $PACKAGE.tar.gz
```
Comment (bimalkjha) [inline .github/workflows/bld_wheels_and_upload.yml:92]: remove v option from tar commands.

File: .github/workflows/bld_wheels_and_upload.yml [hunk 2]
```diff
@@ -75,39 +83,49 @@ jobs:
...
+        run: python -m pip install --upgrade pip build
       - name: Build sdist
         run: python -m build --sdist --no-isolation
+      - name: Build source distribution 
+        run: |
+          cd dist
+          tar -xzvf $PACKAGE.tar.gz 
+          rm -rf $PACKAGE/clidriver*
+          rm -rf $PACKAGE.tar.gz
+          tar -czvf $PACKAGE.tar.gz $PACKAGE
```
Comment (bimalkjha) [inline .github/workflows/bld_wheels_and_upload.yml:95]: @Earammak Here also remove v option, just use -czf

Comment (Earammak) [reply .github/workflows/bld_wheels_and_upload.yml]: Corrected.

File: .github/workflows/bld_wheels_and_upload.yml (hunk 1 above)
Comment (Earammak) [reply .github/workflows/bld_wheels_and_upload.yml]: Corrected.

**Pull request merged**